warehouse/query-core/src/main/java/datawave/query/transformer/ExcerptTransform.java

```diff
@@ -337,84 +347,88 @@ private Set<Excerpt> getExcerpts(PhraseIndexes phraseIndexes) {
      *            the term values to match
      * @return the excerpt
      */
-    private String getExcerpt(String field, int start, int end, Range range, ArrayList<String> hitTermValues) {
+    private Excerpt getExcerpt(String field, int start, int end, Range range, ArrayList<String> hitTermValues) {
```

Comment:
Use the `Excerpt` class which was created for this purpose instead of splitting a returned string elsewhere to get three values.